code/game/gamemodes/changeling/powers/chameleon_skin.dm

```diff
@@ -1,6 +1,6 @@
 /datum/action/changeling/chameleon_skin
 	name = "Chameleon Skin"
-	desc = "Our skin pigmentation rapidly changes to suit our current environment. Costs 25 chemicals."
```

Comment:
It might be better to just use [chemical_cost] for these.

Reply:
Good idea

Reply:
When this was attempted, it did not like chemical cost, so I kept it as numbers, sadly